src/Instance/InstanceDetails/InstanceDetails.js

```diff
@@ -157,20 +147,32 @@ const InstanceDetails = forwardRef(({
     );
   };
 
-  return (
-    <>
-      <Pane
-        {...rest}
-        data-test-instance-details
-        appIcon={<AppIcon app="inventory" iconKey="instance" />}
-        paneTitle={renderPaneTitle()}
-        paneSub={renderPaneSubtitle()}
-        dismissible
-        onClose={onClose}
-        actionMenu={actionMenu}
-        defaultWidth="fill"
-        lastMenu={detailsLastMenu}
-      >
+  const renderDetails = () => {
+    if (isInstanceSharing) {
+      return (
+        <div style={{ paddingTop: '1rem' }}>
```

Comment:
Please use css instead of inline styles

Reply:
Hi @BogdanDenis I removed it at all, this was an old code and looks good without the padding